test/e2e/go.mod

```diff
@@ -206,6 +206,8 @@ require (
 	sigs.k8s.io/controller-runtime v0.20.3
 )
 
-replace go.universe.tf/metallb => github.com/metallb/metallb v0.14.9
-
-replace github.com/coreos/go-iptables => github.com/trozet/go-iptables v0.0.0-20240328221912-077e672b3808
+replace (
+	github.com/coreos/go-iptables => github.com/trozet/go-iptables v0.0.0-20240328221912-077e672b3808
+	github.com/ovn-org/ovn-kubernetes/go-controller => ../../go-controller
```

Comment:
Huh, is this something that you will remove once we merge the api change?

Reply:
Not really. I don't see any other way of consuming this dependency being able to introduce changes and consume them in the same PR. This is what makes sense to me.